pycon/settings.py

```diff
@@ -64,6 +61,7 @@
 LANGUAGES = (
     # disabled italian
     #('it', ugettext('Italiano')),
+    ('es', ugettext('Spanish')),
```

Comment:
ping @oiertwo . I think you've mentioned we needed to support 3 languages, right? Anyway this can go on a following PR..